tests/integration/api_v3/http/dryRunTransactions.test.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { api } = require('../../../helpers/api');
+const { VALID_TRANSACTION, INVALID_TRANSACTION } = require('../constants/dryRunTransactions');
+
+const {
+	badRequestSchema,
+	wrongInputParamSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	goodRequestSchema,
+	dryrunTransactionSchema,
+	metaSchema,
+} = require('../../../schemas/api_v3/dryrunTransaction.schema');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+const endpoint = `${baseUrlV3}/transactions/dryrun`;
+
+describe('Post dryrun transactions API', () => {
```

Comment:
Please apply the corresponding changes requested in `tests/integration/api_v3/rpc/dryRunTransactions.test.js`.